README.md

```diff
@@ -1,37 +1,36 @@
 # python-domino
 
-Python bindings for the Domino API.
+This library provides bindings for the [Domino Data Lab API](https://dominodatalab.github.io/api-docs/).
 
-Permits interaction with a Domino deployment from Python using the [Domino API](https://dominodatalab.github.io/api-docs/).
-
-The latest released version is [1.0.5](https://github.com/dominodatalab/python-domino/archive/1.0.5.zip).
+The latest released version is `1.0.6`.
 
 ## Version Compatibility Matrix
 
 `python-domino` library is compatible with different versions of the `Domino`:
 
-| Domino Versions | Python-Domino |
-| --------------- |:-------------:|
-| 3.6.x or Lower  | [0.3.5](http://github.com/dominodatalab/python-domino/archive/0.3.5.zip)         |
-| 4.1.0 or Higher | [1.0.0](https://github.com/dominodatalab/python-domino/archive/1.0.0.zip) or Higher         |
+| Domino Versions |  Python-Domino  |
+| --------------- |:---------------:|
+| 3.6.x or Lower  | 0.3.5           |
```

Comment:
You removed links to the old version ZIPs, but the old versions are not on PyPy. Is that intended?

Reply:
Well yes, old versions won't be in PyPI. Okay, links probably can go back.